zulipterminal/ui_tools/buttons.py

```diff
@@ -213,6 +215,11 @@ def _narrow_with_compose(self, button: Any) -> None:
         self._view.body.focus.original_widget.set_focus('footer')
         self._view.write_box.private_box_view(self)
 
+    def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
+        if is_command_key('USER_INFO', key):
+            self.model.controller.show_user_info(self.user_id)
```

Comment:
alternatively, can we could just use `self.controller.show_user_info` here , and avoid the line on top? :)